### ISSUE TITLE
Failure reconstruction for USAF 1951 

### ISSUE THREAD

**Issue by yp000925** [farhadnkm/DCOD#1]
Hi Farhad 

Many thanks for your open-source project! 

I successfully reconstruct the cheek sample using the pre-defined parameters setting in the notebook while it seems to fail on the USAF 1951 target after few iterations(as seen in the attached file). Did you use different parameter settings for that reconstruction? 

Looking forward to your reply. 

Thanks 
Sherry
![image](https://user-images.githubusercontent.com/45595057/206827722-06232f84-48cb-4b05-bf9e-326b1588e19e.png)
![image](https://user-images.githubusercontent.com/45595057/206827710-d2dee9db-0295-466e-ba5e-0ac4bcd0d736.png)

**Comment by farhadnkm** [comment]
Hi Sherry,

Yes. That particular sample (as well as the grating one) requires a different set of parameters. I guess they were mentioned in the paper.

Weight decay should be pretty low like 1e-3 or less. You can experiment with hyperparameters e.g., add/remove Gaussian noise for the initializer, try out a lower learning rate, etc.
If problem is not resolved, please let me know to check it out more thoroughly through source code.

Regards,
Farhad

From: ypzhang ***@***.***>
Sent: Saturday, December 10, 2022 4:57 AM
To: farhadnkm/DCOD ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [farhadnkm/DCOD] Failure reconstruction for USAF 1951 (Issue #1)


Hi Farhad

Many thanks for your open-source project!

I successfully reconstruct the cheek sample using the pre-defined parameters setting in the notebook while it seems to fail on the USAF 1951 target after few iterations(as seen in the attached file). Did you use different parameter settings for that reconstruction?

Looking forward to your reply.

Thanks
Sherry
[Image removed by sender. image]<https://user-images.githubusercontent.com/45595057/206827722-06232f84-48cb-4b05-bf9e-326b1588e19e.png>
[Image removed by sender. image]<https://user-images.githubusercontent.com/45595057/206827710-d2dee9db-0295-466e-ba5e-0ac4bcd0d736.png>
